its/projects/WebConfig.CSharp/Web.config

```diff
@@ -0,0 +1,35 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--
```

Comment:
as a note, this has more contents than the VB equivalent? is it intended?

Reply:
Not necessary. I will remove what is not needed.